src/common-utils/devcontainer-feature.json

```diff
@@ -23,36 +23,37 @@
         "username": {
             "type": "string",
             "proposals": [
+                "devcontainer",
                 "vscode",
                 "codespace",
                 "none",
                 "automatic"
             ],
             "default": "automatic",
-            "description": "Enter name of non-root user to configure or none to skip"
+            "description": "Enter name of a non-root user to configure or none to skip"
         },
-        "uid": {
+        "userUid": {
             "type": "string",
             "proposals": [
                 "1000",
                 "automatic"
             ],
             "default": "automatic",
-            "description": "Enter uid for non-root user"
+            "description": "Enter UID for non-root user"
         },
-        "gid": {
+        "userGid": {
```

Comment:
Is a major version upgrade required because of the option name change?

Reply:
Yeah, possibly. What I noticed was that `UID` was set by the OS, so this overlapped and could lead to odd behaviors. When testing, I kept hitting a problem with the UID always being `0` and then I realized what happened.

Reply:
Bumped major to 2.0.0.

Reply:
> What I noticed was that UID was set by the OS, so this overlapped and could lead to odd behaviors. When testing, I kept hitting a problem with the UID always being 0 and then I realized what happened.

Curious, @Chuxel how widespread is this issue? 
I wonder if we'd want to be more caution with bumping major versions? People might get stuck on old versions until they update it. 🤔 

@joshspicer  would soon have make this communication/signaling of needing to bump major versions a bit smoother/clearer. However, until then, I wonder if we could avoid renaming the options and bumping major versions till then? 🤔 

I completely understand if we have to fix the bug if it's widespread, but wondering if we could delay if it's not? 🤔

Reply:
@samruddhikhandale It happened 100% of the time when I was doing testing via `devcontainer feature test`. The UID was always `0` and I got test failures because of it.  I can't speak to its use in the wild... I wouldn't expect most people would set these two properties TBH, so people just may not have noticed.